transport-native-io_uring/src/main/c/netty_io_uring_native.c

```diff
@@ -264,8 +264,13 @@ static jlongArray netty_io_uring_setup(JNIEnv *env, jclass clazz, jint entries)
     memset(&p, 0, sizeof(p));
 
 #ifdef IORING_SETUP_SUBMIT_ALL
-    p.flags = IORING_SETUP_SUBMIT_ALL;
+    p.flags |= IORING_SETUP_SUBMIT_ALL;
 #endif
+    p.flags |= IORING_SETUP_R_DISABLED;
+
+    // TODO: Check if these are supported
```

Comment:
Need to fix this before we pull it in... This needs kernel 6.1

Reply:
Maybe we can do these things in java? How about adding a flag parameter to ioUringSetup?

Reply:
Yes... Just working on it as we speak. I also need this for IORING_SETUP_SUBMIT_ALL.

Reply:
@dreamlike-ocean @franz1981 will use this https://github.com/netty/netty/pull/14677/files#diff-a66727a17a84c1aabaa7d4c343bb227ec6319f9e81ae2500846a9251d3a43794R389

Reply:
done